### ISSUE TITLE
The welcome email format and content needs improvement

### ISSUE THREAD

**Issue by alesanchezr** [refresh-miami/refresh-miami#32]
Email received once users sign up
message " Activate: https://refresh.miami/?page=gf_activation&key=acd78a2213864e09 

![email](https://user-images.githubusercontent.com/82619578/124047458-de8f5c80-d9d9-11eb-86f8-57612b966a4c.PNG)

We expect this email to follow the requirements form #12

**Comment by dimensionmedia** [comment]
I believe this email is different, and I adjusted the formatting on that. I updated Maria on July 6th at 4:29pm via email on this (which is 6 days ago, and this ticket is 10 days - so it's possible wrong site was being tested or this is outdated). I'm attaching a screenshot of the same email screenshot I sent her here.

![Screen Shot 2021-07-06 at 3 42 48 PM](https://user-images.githubusercontent.com/721134/125294563-5f135e80-e2f2-11eb-8fe6-7ef767356bc2.png)
